src/test/java/org/folio/rest/api/migration/StaffSlipsHoldTransitMigrationScriptTest.java

```diff
@@ -0,0 +1,72 @@
+package org.folio.rest.api.migration;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.rest.api.StorageTestSuite;
+import org.folio.rest.support.JsonResponse;
+import org.folio.rest.support.ResponseHandler;
+import org.hamcrest.core.Is;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+public class StaffSlipsHoldTransitMigrationScriptTest extends MigrationTestBase {
```

Comment:
Same question here - if we include these two slips in the existing script, we won't need to create another test class. Though, if we do need to to have a separate script, we can move most of the code of this class to `MigrationTestBase` because it duplicates `StaffSlipsMigrationScriptTest` code.